Bottom: b29f7201270828d53cadbe38613e33f4babfce6d
Top:    180f7c1c55f5decd8030de457d82d654c2d88073
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 16:16:08 +0000

gts: Comment some unused variables left as placeholders

Silences compiler warnings


---

diff --git a/src/gts/pgraph.c b/src/gts/pgraph.c
index 2c13c1e..6d1f619 100644
--- a/src/gts/pgraph.c
+++ b/src/gts/pgraph.c
@@ -30,8 +30,8 @@ static void gnode_split_destroy (GtsObject * object)
     gts_object_destroy (GTS_OBJECT (ns->n));
   }
   else {
-    GtsGNode * n1 = GTS_GNODE_SPLIT_N1 (ns);
-    GtsGNode * n2 = GTS_GNODE_SPLIT_N2 (ns);
+    /* GtsGNode * n1 = GTS_GNODE_SPLIT_N1 (ns); */
+    /* GtsGNode * n2 = GTS_GNODE_SPLIT_N2 (ns); */
 
     g_warning ("Memory deallocation for GtsGNodeSplit not fully implemented yet: memory leak!");
   }
